config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+    big_thumbnails: "Miniature grandi"
+    browse: "sfoglia"
+    cancel: "annulla"
+    cannot_delete_picture_notice: "Impossibile eliminare l'Immagine %{name} poiché è ancora in uso."
+    choose_element_as_target: "Prego scegli un elemento come destinatario"
+    choose_element_to_link: "Prego scegli un elemento"
+    choose_file_to_link: "Prego scegli un file da linkare"
+    "clear clipboard": "svuota appunti"
+    "clear trash": "svuota cestino"
+    click_to_show_all: "Clicca per tornare a mostrare tutto."
+    confirm_to_delete_file: "Vuoi veramente eliminare questo file dal server?"
+    confirm_to_delete_image: "Vuoi veramente eliminare questa immagine dal server?"
+    confirm_to_delete_image_from_server: "Vuoi veramente eliminare questa immagine dal server?"
+    confirm_to_delete_images_from_server: "Vuoi veramente eliminare queste immagini dal server?"
+    confirm_to_delete_page: "Vuoi eliminare veramente questa pagina? I suoi elementi andranno persi!"
+    content_essence_not_found: "Essenza del contenuto non trovata"
+    content_not_found: "Campo per il contenuto non trovato."
+    content_validations_headline: "Prego controlla i campi segnati qui sotto"
+    copy: "copia"
+    copy_element: "Copia questo elemento"
+    copy_page: "Copia pagina"
+    "Could not delete Pictures": "Impossibile eliminare le Immagini"
+    copy_language_tree_heading: "Copia albero delle pagine"
+    country_code_placeholder: 'ad es. us (opzionale)'
+    country_code_foot_note: "Basta che imposti un codice nazione se vuoi fornire ogni nazione con una lingua individuale."
+    create: "crea"
+    "Create language": "Crea una nuova lingua"
+    "Create site": "Crea un nuovo sito"
+    create_language_tree_heading: "Crea un albero delle lingue vuoto"
+    create_page: "Crea una nuova sottopagina"
+    currently_edited_by: "Questa pagina è bloccata da"
+    cut_element: "Taglia questo elemento."
+    delete_file: "Elimina questo file dal server."
+    delete_image: "Elimina quest'immagine"
+    delete_language: "Elimina questa lingua"
+    delete_page: "Elimina questa pagina"
+    delete_tag: 'Elimina tag'
+    document: "File"
+    documents: "File"
+    download_csv: "Scarica CSV"
+    download_file: "Scarica file '%{filename}'"
+    do_you_really_want_to_delete_this_tag?: "Vuoi veramente eliminare questo tag?"
+    drag_to_sort: "Clicca e trascina per riordinare le immagini"
+    edit_file_properties: "Modifica le proprietà del file."
+    edit_image_properties: "Modifica le proprietà dell'immagine."
+    edit_language: "Modifica lingua"
+    edit_page: "Modifica questa pagina"
+    edit_page_properties: "Modifica proprietà pagina"
+    edit_tag: 'Modifica tag'
+    edit_selected_pictures: "Modifica le immagini selezionate"
+    element_editor_not_found: "Errore in questo Elemento"
+    element_of_type: "Elemento"
+    element_saved: "Elemento salvato."
+    enter_external_link: "Prego inserisci l'URL con la quale vuoi creare il link"
```

Comment:
"con il quale" since Locator is masculine gender in Italian (not sure but I think this is the rule here)